### DISCUSSION TITLE
Images not displaying after deploying on vercel

### DISCUSSION THREAD

**Discussion by Patilritik** [vercel/community#6217] [Feature Question] [answered]
### Question

Images not displaying after deploying on vercel but running properly on localhost I added vercel.json also but the issue is not resolved.

**Comment by codewith-vaibhav** [comment]
Hey Bro, Have you got the solution for this issue? I'm facing the same issue post deploying.

**Reply by Patilritik** [reply]
Yes if you are getting the same issue then make sure that path for the image are correct if you are storing the image in public then use the path as '/Images/example.jpg' not  './public/Images/example.jpg'

**Comment by codewith-vaibhav** [comment, marked as answer]
Yep. After a few hours of debugging I finally got the solution. The assets
should be inside the Public folder.
Thanks for the reply btw.

Happy Coding..

**Reply by jj10133** [reply]
I have the same issue I try to run simple plain website. 

Root Dir -> public(images) , src(index.html) but still not showing?

**Reply by Patilritik** [reply]
May be there will be some other problem if you correctly use the path as already mentioned in the above discussion if not then you can show us the code.
